extension/src/shared/ui-components/MyData/utils.ts

```diff
@@ -0,0 +1,26 @@
+import { AddLayerOptions } from "../../../prototypes/layers/states";
+import { MY_DATA_LAYER } from "../../../prototypes/view-layers";
+import { createRootLayerForLayerAtom, RootLayerConfig } from "../../view-layers/rootLayer";
+
+import { UserDataItem } from "./types";
+
+export const handleDataSetSubmit = (
```

Comment:
This is same with this comment: https://github.com/eukarya-inc/PLATEAU-VIEW-3.0/pull/129/files#r1456701098
Also I think this function is used only in MyData component, so you should define this function in MyData component. How do you think?